docs/README.md

```diff
@@ -5,14 +5,58 @@ This web application is built as a [Jekyll](https://jekyllrb.com/) site and is i
 ### Charts
 
 Each `.html` page in this directory contains one or more charts.
-Each chart is defined as an `<svg>` element with a few special attributes that define their behavior:
+Charts are created from `<div>` containers of the class `chart-placeholder` by [`charts.js`](assets/js/charts.js).
+
+#### Structure
+
+Charts generally have the following structure:
+
+```html
+<div class="chart-placeholder">
+	<h3>Title</h3>
+	<canvas
+		data-url="{{ site.dataURL }}/chart-data.tsv"
+		data-type="history"
+		data-config='{...}'
+		></canvas>
+	<div class="info-box">
+		<p>
+			Description of the chart.
+		</p>
+	</div>
+</div>
+```
 
-1. `data-url`: a `.tsv` file generated by the [updater](../updater/README.md) (URL to the data file to be rendered by the chart)
-1. `class`: the kind of chart that will be rendered (`chart`, `table`, or `collaboration`)
-1. `data-type`: the subtype of a chart (`history` or `list`, only applicable if `class` is set to `chart`)
+- `<h3>`: the title of the chart (required)
+- `<canvas>`: used to configure regular charts; change the tag to `<table>` for tables and `<svg>` for collaboration charts
+  - `data-url`: the URL to the `.tsv` file that shall be rendered by the chart (generated by the [updater](../updater/README.md))
+  - `data-type`: the chart type, supported types are:
+    - `history`: time series charts for tracking metrics over time
+    - `list`: horizontal bar charts for general numeric data
+    - `table`: tables directly rendered from the TSV source
+    - `collaboration`: chord diagrams used to visualize connections between entities
```

Comment:
While Hubble's sole chord diagram today is the Collaboration chart, perhaps it would align better to rename this chart type to 'chord'? This way if in the future we ship a _different_ kind of chord diagram, the type would apply a bit more generically?

Reply:
Good point, I’ll change the name to `chord`!